README.md

```diff
@@ -52,37 +52,55 @@ Then, import `SwiftSyntax` in your Swift code.
 ## Contributing
 
 ### Building SwiftSyntax from `master`
+
 Since SwiftSyntax relies on definitions in the main Swift repository to generate the layout of the syntax tree using `gyb`, a checkout of [apple/swift](https://github.com/apple/swift) is still required to build `master` of SwiftSyntax.
 
-To build the `master` version of SwiftSyntax, check `swift-syntax` and  `swift` out side by side
+To build the `master` version of SwiftSyntax, follow the following instructions:
+
+1. Check `swift-syntax` and  `swift` out side by side:
+
 ```
 - (enclosing directory)
   - swift
   - swift-syntax
 ```
 
-And run `swift-syntax/build-script.py`. SwiftSyntax is now being built with the Swift compiler installed on the system.
+2. Make sure you have a recent [master Swift toolchain](https://swift.org/download/#snapshots) installed.
+3. Define the `TOOLCHAINS` environment variable as below to have the `swift` command point inside the toolchain:
+
+```
+$ export TOOLCHAINS=swift
```

Comment:
What happens if we have multiple OSS toolchain in the dir. Will this select the latest master?

Reply:
It will

Reply:
I've noticed if any swift-5.0-development snapshots are present, these toolchains will take precedence over the master ones. Could you mention this caveat?

Reply:
Sure!

Reply:
I updated the documentation to warn against version specific toolchains. Let me know if you think this is enough.